.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```

Comment:
- [Optional] Please set up linters for javascript by copying the files and folders from [this](https://github.com/microverseinc/linters-config/tree/master/javascript) repo to your own repo and run `npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x` to install eslint.